### DISCUSSION TITLE
Change versioning from SemVer to CalVer

### DISCUSSION THREAD

**Discussion by elcajon** [homeassistant-apps/app-cloudflared#728] [Ideas]
@brenner-tobias, just an idea....

As the add-on is stable and few changes or new features are added, most releases tend to be patches or dependency updates. To make it easier to see when the last release was made, I think it would be a good time to switch from semantic to calendar versioning. Maybe at the turn of the year? 
I did the same with my code-server add-on (albeit mostly for private use) [without having to make any major changes to the CI](https://github.com/elcajon-dev/addon-code-server/pull/670/files).

**Comment by brenner-tobias** [comment]
Thanks a lot for the suggestion, though I am not convinced why we should change to be honest.
All HA add-ons and community add-ons use SemVer and though we are pretty stable, in case we do implement more functionality if not breaking changes, it is easily spotted in SemVer, while CalVer focuses on the time, which I do not think is that relevant, but rather what the level of change was.

I do acknowledge that this is also a personal preference and I am happy to consider CalVer if there are more reasons from your perspective, so happy to have a discussion :-)

**Reply by elcajon** [reply]
I actually agree with you, the only advantage of CalVer is that you can see at a glance in HA that you have a relatively up-to-date version installed, whereas with 5.1.32 you don't know when it's from.
I'm not attached to it, for me it's just a personal preference too.
